Add TYPO3 Unit tests to the OSS framework parity check

tmotyl opened the pull request:
Tested on Centos 6.4 and
HipHop VM 3.5.0-dev (rel)
Compiler: heads/master-0-g7024026e00dec2fe6ca5ea46e23e5d5181648eef
Repo schema: 75a53eceff2c3914f29531368d9d363bf9507a35
Extension API: 20140829

fixes: https://github.com/facebook/hhvm/issues/2103

GrahamCampbell commented:
You need to actually run the tests and commit the recorded results.

GrahamCampbell commented on hphp/test/frameworks/frameworks.yaml:
```diff
@@ -632,6 +632,36 @@
+  typo3:
+    url: git://github.com/TYPO3/TYPO3.CMS.git
+    branch: master
+    commit: 42ccc0ce1850f3b2f9c7a01efe49230687bf579a
```
if you're not using the latest tagged release, you need to include a comment explaining why

tmotyl commented:
added files with results
currently it is: 
date,               typo3
2015/01/11-21:47:47,99

And changed TYPO3 version to the current stable 6.2 LTS instead of master
